addons/l10n_tr/data/template/account.tax-tr.csv

```diff
@@ -1,17 +1,112 @@
-"","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","tax","refund","tr191",""
-"tr_kdv_satis_sale_20","13","","VAT 20%","20%","False","20.0","percent","sale","tax_group_kdv_20","base","invoice","","KDV %20(Satış)"
-"","","","","","","","","","","tax","invoice","tr391",""
-"","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","tax","refund","tr191",""
-"tr_kdv_satis_purchase_20","14","","VAT 20%","20%","False","20.0","percent","purchase","tax_group_kdv_20","base","invoice","","KDV %20(Alım)"
-"","","","","","","","","","","tax","invoice","tr391",""
-"","","","","","","","","","","base","refund","",""
-"","","","","","","","","","","tax","refund","tr191",""
+"id","sequence","description","invoice_label","name","price_include","amount","amount_type","children_tax_ids","type_tax_use","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/tag_ids","description@tr"
+"tr_ers_n20","1","Export Registered Sales VAT (-20%)","-20%","EX RS -20%","False","-20","percent","","none","tr_tax_group_kdv_n20","base","invoice","","","İhracat Kayıtlı Satış KDV'si (-20%)"
+"","","","","","","","","","","","tax","invoice","tr391004","",""
+"","","","","","","","","","","","base","refund","","",""
+"","","","","","","","","","","","tax","refund","tr391004","",""
+"tr_s_wh_20_2_10","2","Sales VAT with Withholding Tax 20% (2/10)","","WH 20% (2/10)","False","","group","tr_wh_20_2_10,tr_s_20","sale","tr_tax_group_kdv_20","","","","","Stopajlı Satış KDV'si %20 (2/10)"
+"tr_ers_20","3","Export Registered Sales VAT 20%","","EX RS 20%","False","","group","tr_ers_n20,tr_s_20","sale","tr_tax_group_kdv_20","","","","","İhracat Kayıtlı Satış KDV'si %20"
+"tr_wh_20_5_10","4","Withheld VAT 20% (Withholding 5/10)","-10%","WH 20% (5/10)","False","-10","percent","","none","tr_tax_group_wh_10","base","invoice","","+SAL_20_BASE_5_10","Tevkif KDV %20 (Tevkifat 5/10)"
+"","","","","","","","","","","","tax","invoice","tr391004","-SAL_20_TAX_5_10",""
+"","","","","","","","","","","","base","refund","","-SAL_20_BASE_5_10",""
+"","","","","","","","","","","","tax","refund","tr391004","+SAL_20_TAX_5_10",""
+"tr_s_wh_20_5_10","5","Sales VAT with Withholding Tax 20% (5/10)","","WH 20% (5/10)","False","","group","tr_wh_20_5_10,tr_s_20","sale","tr_tax_group_kdv_20","","","","","Stopajlı Satış KDV'si %20 (5/10)"
+"tr_wh_s_20","6","Withholding Sales Tax 20%","-20%","WH 20%","False","-20","percent","","sale","tr_tax_group_wh_20","base","invoice","","","İhracat Kayıtlı Satış KDV'si (-20%)"
+"","","","","","","","","","","","tax","invoice","tr391001","",""
+"","","","","","","","","","","","base","refund","","-SAL_BASE_20",""
+"","","","","","","","","","","","tax","refund","tr391001","-SAL_TAX_20",""
+"tr_p_0","7","Purchase 0%","0%","PUR 0%","False","0","percent","","purchase","tr_tax_group_kdv_0","base","invoice","","+PUR_BASE_0","%0 satın al"

```

Comment:
```suggestion
"tr_p_0","7","Purchase 0%","0%","0%","False","0","percent","","purchase","tr_tax_group_kdv_0","base","invoice","","+PUR_BASE_0","%0 satın al"

```
No need to specify it's a purchase tax, it will only be available for purchase anyway

Reply:
I thought I removed PUR :sweat_smile: